diff -ru Montage-master/Montage/mDiffExec.c Montage-master-patched/Montage/mDiffExec.c
--- Montage-master/Montage/mDiffExec.c	2020-07-28 16:28:07.000000000 +0200
+++ Montage-master-patched/Montage/mDiffExec.c	2020-11-06 23:23:17.487945200 +0100
@@ -315,12 +315,21 @@
 
       if(strcmp( status, "ABORT") == 0)
       {
-	 strcpy( msg, svc_value( "msg" ));
+         strcpy(msg, svc_value("msg"));
 
-	 fprintf(fstatus, "[struct stat=\"ERROR\", msg=\"%s\"]\n", msg);
-	 fflush(stdout);
-
-	 exit(1);
+         fprintf(fstatus, "[struct stat=\"ERROR\", msg=\"%s\"]\n", msg);
+         fflush(stdout);
+         /* some errors from mDiff are ignored, such as images not overlapping */
+         if (strcmp(msg, "Images don't overlap") == 0 || strcmp(msg, "All pixels are blank.") == 0)
+         {
+            /* ensure the output file exists, even if it is 0 bytes */
+            fopen(output_file, "w+");
+            exit(0);
+         }
+         else
+         {
+            exit(1);
+         }
       }
 
       ++count;
