--- Montage-master/MontageLib/Project/montageProject.c	2020-07-28 16:28:07.000000000 +0200
+++ Montage-patched/MontageLib/Project/montageProject.c	2020-11-07 13:14:24.696072290 +0100
@@ -856,6 +856,9 @@
    if(oxpixMin > oxpixMax || oypixMin > oypixMax)
    {
       sprintf(returnStruct->msg, "No overlap");
+      returnStruct->status = 0;
+      fopen(output_file, "w+");
+      fopen(area_file, "w+");
       return returnStruct;
    }
     
--- Montage-master/MontageLib/ProjectPP/montageProjectPP.c	2020-07-28 16:28:07.000000000 +0200
+++ Montage-patched/MontageLib/ProjectPP/montageProjectPP.c	2020-11-07 13:15:56.911300195 +0100
@@ -807,6 +807,9 @@
    {
       mProjectPP_printError("No overlap");
       strcpy(returnStruct->msg, montage_msgstr);
+      returnStruct->status = 0;
+      fopen(output_file, "w+");
+      fopen(area_file, "w+");
       return returnStruct;
    }
     
--- Montage-master/MontageLib/ProjectCube/montageProjectCube.c	2020-07-28 16:28:07.000000000 +0200
+++ Montage-patched/MontageLib/ProjectCube/montageProjectCube.c	2020-11-07 13:14:59.868853994 +0100
@@ -821,6 +821,9 @@
    if(oxpixMin > oxpixMax || oypixMin > oypixMax)
    {
       sprintf(returnStruct->msg, "No overlap");
+      returnStruct->status = 0;
+      fopen(output_file, "w+");
+      fopen(area_file, "w+");
       return returnStruct;
    }
     
